upup/models/cloudup/resources/addons/aws-ebs-csi-driver.addons.k8s.io/k8s-1.17.yaml.template

```diff
@@ -478,6 +478,13 @@ spec:
             - "--extra-tags={{ CloudLabels }}"
             - --http-endpoint=0.0.0.0:3301
             - --v=5
+            {{- $tags := list }}
+            {{- range $key, $value := .ExtraVolumeTags }}
+            {{- $tags = append $tags (printf "%s=%v" $key $value) }}
+            {{- end }}
+            {{- if gt (len $tags) 0 }}
+            {{ printf "- \"--extra-tags=%s\"" (join "," $tags) }}
```

Comment:
@heybronson Any thought about just using `CloudLabels`?
See  line 478 above:
```
- "--extra-tags={{ CloudLabels }}"
```

Reply:
Because of the IAM policies it's also quite important that `CloudLabels` are unconditionally added.

Reply:
I like the idea of just using `CloudLabels`; that's the primary use case I was targeting anyways. Makes more sense to use the existing `CloudLabels` instead of defining them twice.